low-level/files/opt/seagate/sspl/setup/resource_map/resource_map.py

```diff
@@ -67,3 +67,23 @@ def get_health_template(uid, is_fru: bool):
                 "specifics": []
             }
         }
```

Comment:
I couldn't comment to line 62.
So please consider this comment for that.
```
"last_updated": int(time.time()),
```
Also ensure `import time` is added in this module top.

Reply:
Done.

Reply:
As others suggest,  `last_updated `can be set in set_health_data() and that is more suitable.

Reply:
Done.